capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -752,6 +760,38 @@ private void bindViews(final View view) {
         mUnsupportedQRCodeDetectedPopupContainer = view.findViewById(
                 R.id.gc_unsupported_qrcode_detected_popup_container);
         mImageStack = view.findViewById(R.id.gc_image_stack);
+        topAdapterInjectedViewContainer = view.findViewById(R.id.gc_navigation_top_bar);
+
+    }
+
+    private void setTopBarInjectedViewContainer() {
+        if (GiniCapture.hasInstance()) {
+            topAdapterInjectedViewContainer.setInjectedViewAdapter(GiniCapture.getInstance().getNavigationBarTopAdapter());
+
+            if (topAdapterInjectedViewContainer.getInjectedViewAdapter() == null)
+                return;
+
+            if (mFragment.getActivity() == null)
+                return;
+
+            topAdapterInjectedViewContainer.getInjectedViewAdapter().setNavButtonType(NavButtonType.BACK);
+            topAdapterInjectedViewContainer.getInjectedViewAdapter().setTitle(mFragment.getActivity().getResources().getString(R.string.gc_title_camera));
+            topAdapterInjectedViewContainer.getInjectedViewAdapter().setMenuResource(R.menu.gc_camera);
+            topAdapterInjectedViewContainer.getInjectedViewAdapter().setOnMenuItemClickListener(item -> {
+                startHelpActivity();
```

Comment:
[Request] Please check the click item's id and only start help activity, if it's the right id. If it's an unknown id, then I propose to throw an `UnsupportedOperationException` with a message that says something like "Unknown menu item id. Please don't call our OnMenutItemClickListener for custom menu items.".

This way we prevent clients calling our `OnMenuItemClickListener` for their custom menu items. If they use custom menu items, then they should call our click listener only for the menu items we provided. Dealing with their custom menu items is not our problem. 🙂

Reply:
Done